api_app/analyzers_manager/file_analyzers/detectiteasy.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, FileAnalyzer
+from tests.mock_utils import MockUpResponse
+
+logger = logging.getLogger(__name__)
+
+
+class DetectItEasy(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "executable_analyzer"
+    url: str = "http://malware_tools_analyzers:4002/die"
+    # http request polling max number of tries
+    max_tries: int = 10
```

Comment:
the combination of max tries * polling retries should be the same as the value of the soft time limit, otherwise it does not male sense